.travis/tests/terraform.sh

```diff
@@ -32,9 +31,4 @@ go get github.com/mattn/goveralls
 make build
 
 # Run tests
-cd opennebula && go test -coverprofile=profile.out -v
-if [ -f profile.out ]; then
-    $(go env GOPATH | awk 'BEGIN{FS=":"} {print $1}')/bin/goveralls -coverprofile=profile.out -service=travis-ci -repotoken $COVERALLS_TOKEN
-    go tool cover -func=profile.out
-    rm profile.out
-fi
+make testacc
```

Comment:
Line 28 is not removed like other test coverage stuffs ?
go get github.com/mattn/goveralls

You add make testacc.
It's a bit more that removing coverage as the commit message explain, in addition, will the test result state be included under the build marquer ?